src/SPC/builder/Extension.php

```diff
@@ -448,6 +449,15 @@ public function buildUnixShared(): void
             ->exec('make clean')
             ->exec('make -j' . $this->builder->concurrency)
             ->exec('make install');
+
+        // process *.so file
+        $soFile = BUILD_MODULES_PATH . '/' . $this->getName() . '.so';
+        if (!file_exists($soFile)) {
+            throw new ValidationException("extension {$this->getName()} build failed: {$soFile} not found", validation_module: "Extension {$this->getName()} build");
+        }
+        /** @var UnixBuilderBase $builder */
+        $builder = $this->builder;
+        $builder->deployBinary($soFile, $soFile, false);
```

Comment:
@henderkes Here and `LinuxBuilder.php:284` shows it will call extractDebugInfo() already. See: `UnixBuilderBase.php:142`.